### PR TITLE
feat: add a shortcut to the Start menu

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add a shortcut to aviutl.exe to the Start menu.

**This PR depends on the following PR(s).**
- #196

## Motivation and Context

When you install other video editing software, you will see a shortcut in the start menu. However, this is not the case with AviUtl. I think this is one of the reasons why beginners find AviUtl difficult to use.

## How Has This Been Tested?

- When you install AviUtl, a shortcut will be created.
- If you select a folder with AviUtl, a shortcut will be created.
- If you select a folder without AviUtl, the shortcut will be removed.
- If you uninstall apm, the shortcut will also be removed.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/core/core.js [hunk 1]
```diff
@@ -71,6 +93,19 @@ async function displayInstalledVersion(instPath) {
...
     replaceText('core-check-date', '未確認');
   }
+
+  // Add a shortcut to the Start menu
+  if (process.platform === 'win32') {
+    const appDataPath = await ipcRenderer.invoke('app-get-path', 'appData');
+    const aviutlPath = path.join(instPath, 'aviutl.exe');
+    if (fs.existsSync(aviutlPath)) {
+      shortcut.addAviUtlShortcut(appDataPath, aviutlPath);
+      console.log('add');
```
Comment (hal-shu-sato) [inline src/core/core.js:103]: Is this forgotten to remove?

**Review by hal-shu-sato — COMMENTED** [review]
Good feature. It works.
But I have some questions.

File: src/core/core.js [hunk 2]
```diff
@@ -71,6 +93,19 @@ async function displayInstalledVersion(instPath) {
...
+    } else {
+      shortcut.removeAviUtlShortcut(appDataPath);
+      console.log('remove');
```
Comment (hal-shu-sato) [inline src/core/core.js:106]: Is this also forgotten to remove?

**Comment by mitosagi** [comment]
I fixed it.

**Review by hal-shu-sato — APPROVED** [review]
Thanks!

**Pull request merged**